dockerfiles/default-configs/users.yaml

```diff
@@ -0,0 +1,24 @@
+---
+roles:
```

Comment:
I think, the default user config is not the best one. I suggest to add `read` access to `writer`. 
`users` should be updated to have common user names: `admin`, `readonly`, `user`. 
For default config it is better to unify all passwords and set them to `password`.

Please, update `users.yaml` in the same way in `config_examples` too